cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -161,20 +161,22 @@ def check_ftp(appliance, ftp, server_name, server_zone_id, check_ansible_logs=Fa
         try:
             date_from = datetime.strptime(date_from, "%Y%m%d%H%M%S")
             date_to = datetime.strptime(date_to, "%Y%m%d%H%M%S")
-            logs_ansible = "log/ansible_tower"
-            if ftp.login != 'anonymous' and check_ansible_logs:  # can't login as anon using SSH
+            log_files = ["log/ansible_tower", "pgsql/data/postgresql.conf"]
+            if ftp.login != 'anonymous' and check_contents:  # can't login as anon using SSH
                 with SSHClient(hostname=ftp.host,
                                username=ftp.login,
                                password=ftp.password) as log_ssh:
-                    result = log_ssh.run_command(
-                        "unzip -l ~{} | grep {}".format(zip_file.path, logs_ansible),
-                        ensure_user=True)
-                    assert '.log' in result.output
-                    log_file_size = result.output.split()[0]
-                    assert int(log_file_size) > 0, "Log file is empty!"
-
+                    for log in log_files:
+                        if "ansible" in log and BZ(1751961).blocks:
+                            continue
+                        result = log_ssh.run_command(
+                            "unzip -l ~{} | grep {}".format(zip_file.path, log), ensure_user=True
+                        )
+                        assert log in result.output
+                        log_file_size = result.output.split()[0]
+                        assert int(log_file_size) > 0, "Log file is empty!"
         except ValueError:
-            assert False, "Wrong file matching of {}".format(zip_file.name)
+            assert False, "Wrong file matching in {} for {}".format(zip_file.name, log)
```

Comment:
@lina-nikiforova Could you update this line to use `f strings`

Reply:
@jawatts sure, pushed :)